pkg/daemon/rpm-ostree.go

```diff
@@ -131,18 +185,29 @@ func (r *RpmOstreeClient) Rebase(container, osImageContentDir string) (changed b
 	}
 
 	var imageData *types.ImageInspectInfo
-	imageData, err = imageInspect(container)
-	if err != nil {
-		return
+	if imageData, err = imageInspect(imgURL); err != nil {
+		if err != nil {
+			var podmanImgData *imageInspection
+			glog.Infof("Falling back to using podman inspect")
+			if podmanImgData, err = podmanInspect(imgURL); err != nil {
+				return
+			}
+			ostreeCsum = podmanImgData.Labels["com.coreos.ostree-commit"]
+			ostreeVersion = podmanImgData.Labels["version"]
+		}
+	} else {
+		ostreeCsum = imageData.Labels["com.coreos.ostree-commit"]
+		ostreeVersion = imageData.Labels["version"]
 	}
+	// We may have pulled in OSContainer image as fallback during podmanCopy() or podmanInspect()
+	defer exec.Command("podman", "rmi", imgURL).Run()
```

Comment:
should this be `podman rmi -f` to make it idempotent in case we haven't pulled it?

Reply:
+1, forcing is good to do here.